components/autogen/src/format-pages.txt

```diff
@@ -453,10 +454,11 @@ writer = CellH5Writer
 mif = true
 
 [Cellomics]
-extensions = .c01
+extensions = .c01, .dib
```

Comment:
This change should also be propagated to `supported-formats.txt` as well as the specific format page.